src/features/bookings/components/EndedBookingItem.native.test.tsx

```diff
@@ -158,14 +196,30 @@ describe('EndedBookingItem', () => {
   })
 
   describe('when reaction feature flag is activated', () => {
-    beforeEach(() => {
-      useFeatureFlagSpy.mockReturnValueOnce(true)
+    beforeAll(() => {
+      useFeatureFlagSpy.mockReturnValue(true)
+      mockGetConfigValues.mockReturnValue({
+        reactionCategories: { categories: ['SEANCES_DE_CINEMA'] },
+      })
+    })
+
+    it('should display reaction button', async () => {
+      renderEndedBookingItem(bookingsSnap.ended_bookings[1], RemoteConfigProvider)
+
+      await waitFor(() =>
+        expect(screen.getByLabelText('Réagis à ta réservation')).toBeOnTheScreen()
+      )
     })
 
-    it('should display reaction button', () => {
-      renderEndedBookingItem(bookingsSnap.ended_bookings[1])
+    it('should not display reaction button when category is not in remoteConfig param', async () => {
+      mockGetConfigValues.mockReturnValueOnce({
+        reactionCategories: { categories: ['THEATRE'] },
+      })
+      renderEndedBookingItem(bookingsSnap.ended_bookings[1], RemoteConfigProvider)
 
-      expect(screen.getByLabelText('Réagis à ta réservation')).toBeOnTheScreen()
+      await waitFor(() =>
```

Comment:
C'est bizarre d'avoir un `waitFor` sur un `not` tu peux vérifier que ta page est bien chargée en faisant ça :

```
      await screen.findByText('Réservation utilisée')

      expect(screen.queryByLabelText('Réagis à ta réservation')).not.toBeOnTheScreen()
```

Reply:
Voilà c'est corrigé